### ISSUE TITLE
ids=all gets parsed as a list and does not allow returning all 

### ISSUE THREAD

**Issue by NoOneAtHome** [JuxhinDB/gw2-api-interface#17]
Currency, https://wiki.guildwars2.com/wiki/API:2/currencies and Materials https://wiki.guildwars2.com/wiki/API:2/materials can use ids=all e.g. https://wiki.guildwars2.com/wiki/API:2/currencies?ids=all  this returns the full list of those calls.  If you set ids=all it is getting set to a list and is actually sending ids=a,l,l which of course does not work.  

I have fixed it locally by modifying gw2api/objects/base_object.py at line 76:

```
        if _id:
            request_url += f'id={str(_id)}&'  # {base_url}/{object}/{id}
        elif ids:
            try:
                if ids=='all':
                    request_url += 'ids=all&'
                else:
                    request_url += 'ids=' + ','.join([str(_) for _ in ids]) + '&'
            except TypeError:
                print("Could not add ids because the given ids argument is not an iterable.")
```

**Comment by JuxhinDB** [comment]
Good catch, looks like you're pretty much there with a solution. Would you like to open a PR and I can guide you through it?

**Comment by JuxhinDB** [comment]
Merged and bumped version 0.7.0 for you. Let me know if you have any issues. Thanks again
